mqtt/mqtt-broker/src/persist.rs

```diff
@@ -136,6 +145,211 @@ impl FileFormat for BincodeFormat {
     }
 }
 
+#[derive(Clone, Debug, Default)]
+pub struct ConsolidatedStateFormat;
+
+impl ConsolidatedStateFormat {
```

Comment:
I see that you did similar to FileFormat, but instead you can auto implement it with `#[derive(Default)]` feature